bedrock/firefox/templates/firefox/lockwise/lockwise.html

```diff
@@ -0,0 +1,105 @@
+{% add_lang_files "firefox/products/lockwise" %}
+
+{% from "macros.html" import google_play_button with context %}
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_favicon %}{{ static('img/firefox/lockwise/favicon.ico') }}{% endblock %}
+
+{% block page_title %}{{_('Firefox Lockwise — password manager —  take your passwords everywhere')}}{% endblock %}
+
+{% block page_desc %}{{ _('Firefox Lockwise lets you securely access the passwords you’ve saved in Firefox from anywhere — even outside of the browser. Features 256-bit encryption and Face/Touch ID.') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('lockwise') }}
+{% endblock %}
+
+{% block body_id %}firefox-lockwise{% endblock %}
+{% block body_class %}firefox-lockwise{% endblock %}
+
+{% block content %}
+  <main role="main">
+    <section class="mzp-c-hero mzp-has-image">
+      <div class="mzp-l-content">
+        <div class="mzp-c-hero-body">
+          <img src="{{ static('img/firefox/lockwise/lockwise-horizontal.svg') }}" class="lockwise-logo" />
+          <h1 class="mzp-c-hero-title">{{ _('Take your passwords everywhere') }}</h1>
+          <div class="mzp-c-hero-desc">
+            <p>{{ _('Securely access the passwords you’ve saved in Firefox from anywhere — even outside of the browser.') }}</p>
+          </div>
+          <div class="mobile-download-buttons">
+            <a href="{{ lockwise_adjust_url('ios', 'accounts-page') }}" data-link-type="download" data-download-os="iOS" class="ios-button"  data-cta-text="AppleAppStore" data-cta-type="Lockwise-AppStore" data-cta-position="Primary">
+              <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+            </a>
+            {{ google_play_button(href=lockwise_adjust_url('android', 'accounts-page'), extra_data_attributes={'download-location': button_location, 'cta-type': 'Lockwise-AppStore', 'cta-text': 'GooglePlayStore', 'cta-position': 'Primary' } ) }}
+          </div>
+          <div class="mzp-c-hero-cta add-on-download for-firefox-69-and-below" >
+            <div class="mzp-c-button-download-container">
+              <p >{{ _('Try Lockwise now') }}</p>
+              <a href="https://github.com/mozilla-lockwise/lockwise-addon/releases/download/2.2.5-alpha/lockwise-signed.xpi" class="mzp-c-button mzp-t-product" data-cta-type="Lockwise-Install" data-cta-position="Primary">{{ _('Install for Firefox') }}</a>
+            </div>
+          </div>
+          <div class="mzp-c-hero-cta add-on-download for-firefox-70-and-above" >
+            <div class="mzp-c-button-download-container">
+              <p >{{ _('Try Lockwise now') }}</p>
```

Comment:
This is coming through as untranslated when I view it in French. I don't see this string in the .lang file? https://github.com/mozilla-l10n/www.mozilla.org/blob/master/en-US/firefox/products/lockwise.lang

Reply:
yes there are a few strings that got added after I did the initial string pulling. I'm working with Eric to get it added, but pages won't go live until they are there.